sys/Makefile

```diff
@@ -140,7 +140,6 @@ ifneq (,$(filter cord_ep,$(USEMODULE)))
     DIRS += net/application_layer/cord/ep
 endif
 
```

Comment:
this is unrelated but I think we can live with it. ;)

Reply:
Sorry, small mistake...